applications/serial_lte_modem/doc/PPP_AT_commands.rst

```diff
@@ -93,3 +92,26 @@ Example
   #XPPP: 0,0
 
   OK
+
+Testing on Linux
+================
+
+You can test SLM's PPP on Linux by using the ``pppd`` command.
+Having PPP running on SLM and its UART connected to the Linux host, run the following command:
+
+   .. code-block:: console
+
+      $ sudo pppd -detach <PPP_UART_dev> <baud_rate> noauth crtscts novj nodeflate nobsdcomp debug +ipv6 usepeerdns noipdefault defaultroute defaultroute6 ipv6cp-restart 5 ipcp-restart 5
+
+Make sure to replace ``<PPP_UART_dev>`` by the device file assigned to the PPP UART (e.g. :file:`/dev/ttyACM1`) and ``<baud_rate>`` by the baud rate of the UART that PPP is using.
+
+After running the command and the PPP link negotiation completes successfully, a new network interface will be created (typically ``ppp0``); it will send and receive IP traffic through the modem of the nRF91 Series SiP running SLM.
+
+.. note::
+
+   You may encounter some issues with DNS resolution.
+   They can be worked around by manually editing :file:`/etc/resolv.conf`.
+   You can put in DNS servers that are reachable with your current network configuration.
+
+   Those can even be the DNS servers that SLM's PPP sends as part of the PPP link negotiation.
+   They are the DNS servers of the default PDN connection obtained from the modem.
```

Comment:
```suggestion
   These added servers can even be the DNS servers that SLM's PPP sends as part of the PPP link negotiation, which are the DNS servers of the default PDN connection obtained from the modem.
```
Note the removed empty line so we don't end up with a one-sentence paragraph.

Reply:
Is it forbidden? :smiley: (Curious to know the actual rationale.)

Reply:
Not forbidden, but it's generally not good form in writing. In the same way as we try to avoid really long-running sentences, but sometimes they are required (like here I actually made one by combining two shorter sentences).

We still use one-sentence paragraphs as introductions to procedure steps and such, and sometimes for emphasis or visibility.